index.js

```diff
@@ -146,15 +146,19 @@ module.exports = app => {
     const customGeneratedFiles = await getCustomGeneratedFiles(context, owner, repo)
     const customLabels = await context.config('labels.yml', labels)
 
-    // list of files modified in the pull request
-    const res = await context.octokit.pulls.listFiles({
+    // grab all pages
+    const files = await context.octokit.paginate(
```

Comment:
can you please add `?per_page=100` to wherevever you see `.get(baseURL+'/pulls/${pull_number}/files')` in test.js to get the tests working